packages/panzer/mini-em/src/closures/MiniEM_DarcyAnalyticForcing_impl.hpp

```diff
@@ -53,8 +53,7 @@ template <typename EvalT,typename Traits>
 void DarcyAnalyticForcing<EvalT,Traits>::evaluateFields(typename Traits::EvalData workset)
 {
   const double time = workset.time;
-  const double pi = Kokkos::Experimental::pi_v<double>;
-  // const double pi = Kokkos::numbers::pi_v<double>;
+  const double pi = Kokkos::pi_v<double>;
```

Comment:
```suggestion
  const auto pi = Kokkos::numbers::pi_v<double>;
```

Reply:
Thanks, fixed in [b15d9ac](https://github.com/trilinos/Trilinos/pull/11707/commits/b15d9acaf3a6910709b2ce2eee5d6368f7fea4e2)